skimage/filters/thresholding.py

```diff
@@ -200,8 +200,10 @@ def threshold_isodata(image, nbins=256, return_all=False):
 
     Histogram-based threshold, known as Ridler-Calvard method or inter-means.
     Threshold values returned satisfy the following equality:
```

Comment:
For block syntax, you need to indent the content of the block. That's probably why it did not work?